client/src/components/PortfolioAlerts.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import { Trans } from "@lingui/macro";
+import classNames from "classnames";
+import { Link, useLocation } from "react-router-dom";
+import networkDiagram from "../assets/img/network-diagram.png";
+import { AddressPageRoutes } from "../routes";
+import { Alert, AlertProps } from "./Alert";
+import { logAmplitudeEvent } from "./Amplitude";
+import Modal from "./Modal";
+import { isLegacyPath } from "./WowzaToggle";
+
+export const BIG_PORTFOLIO_THRESHOLD = 300;
+export const FILTER_PORTFOLIO_THRESHOLD = 15;
+
+type PortfolioAlertProps = Omit<AlertProps, "children"> & {
+  portfolioSize: number;
+};
+
+type FilterPortfolioAlertProps = PortfolioAlertProps & {
+  addressPageRoutes: AddressPageRoutes;
+};
+
+export const BigPortfolioAlert = ({ portfolioSize, className, ...props }: PortfolioAlertProps) => {
```

Comment:
alerts would be an excellent candidate for making into a reusable class component in storybook later! esp since these exact patterns also exist in LOC rebrand

Reply:
Yeah for sure! That was the next one I was hoping to add, it should be mostly ready to copy over from here and just the styles need to be updated.